expyfun/stimuli/tests/test_stimuli.py

```diff
@@ -31,22 +31,36 @@ def test_textures():
 def test_hrtf_convolution():
     """Test HRTF convolution."""
     data = np.random.randn(2, 10000)
-    assert_raises(ValueError, convolve_hrtf, data, 44100, 0)
+    assert_raises(ValueError, convolve_hrtf, data, 44100, 0, interp=False)
     data = data[0]
-    assert_raises(ValueError, convolve_hrtf, data, 44100, 0.5)  # invalid angle
-    for source in ['barb', 'cipic']:
-        out = convolve_hrtf(data, 44100, 0, source=source)
-        out_2 = convolve_hrtf(data, 24414, 0, source=source)
-        assert_equal(out.ndim, 2)
-        assert_equal(out.shape[0], 2)
-        assert_true(out.shape[1] > data.size)
-        assert_true(out_2.shape[1] < out.shape[1])
-        # ensure that, at least for zero degrees, it's close
-        out = convolve_hrtf(data, 44100, 0, source=source)[:, 1024:-1024]
-        assert_allclose(np.mean(rms(out)), rms(data), rtol=1e-1)
-        out = convolve_hrtf(data, 44100, -90, source=source)
-        rmss = rms(out)
-        assert_true(rmss[0] > 4 * rmss[1])
+    assert_raises(ValueError, convolve_hrtf, data, 44100, 0.5, interp=False)
+    assert_raises(ValueError, convolve_hrtf, data, 44100, 0, source='other',
+                  interp=False)
+    assert_raises(ValueError, convolve_hrtf, data, 44100, 185, interp=True)
+    assert_raises(ValueError, convolve_hrtf, data, 44100, 0, interp='other')
```

Comment:
@maddycapp27 for these `assert_raises` calls, I find it helpful to have either comments explaining what is wrong (cf. the `# invalid angle` comment that used to be here), or alternatively, make it really obvious by the function parameters themselves which one is the invalid one (e.g., by using arguments like `interp='foo'` or `interp='bogus_arg'`).